packages/opentelemetry-core/package.json

```diff
@@ -71,6 +71,7 @@
     "karma-mocha": "2.0.1",
     "karma-spec-reporter": "0.0.32",
     "karma-webpack": "4.0.2",
+    "lerna": "3.22.1",
```

Comment:
why does lerna need to be in dev deps ?

Reply:
he doesn't want to have to have it installed globally. The npm scripts call lerna so if you have it in dev deps they find it correctly. If not, it looks for global.

Reply:
Yeah but i meant why does it need to be commited ? We expect it to be available globally right ?

Reply:
If you dont have lerna locally you cant compile it. Not mentioning that the lerna we are using should be compatible with the one that is in the project. This is also missing in other packages which should be added. We should not expect user to have lerna installed globally

Reply:
Then do we need to add it to all other packages ? I would prefer to do this in another PR